feature-swap-impl/src/main/res/layout/fragment_main_swap_settings.xml

```diff
@@ -90,14 +90,24 @@
         android:layout_marginTop="8dp"
         app:layout_constraintTop_toBottomOf="@id/swapMainSettingsReceiveTitle" />
 
+    <io.novafoundation.nova.common.view.AlertView
+        android:id="@+id/swapMainSettingsAlert"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/swap_error_pair_not_supported"
+        app:layout_constraintTop_toBottomOf="@+id/swapMainSettingsReceiveInput"
+        app:style="error" />
```

Comment:
We don't need error alert anymore in the updated design. We will only have field validations + info alert
<img width="806" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/70131744/07bda45e-c4e9-4eba-b1db-614f6d1d35d9">
<img width="418" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/70131744/82b4d319-8b07-4d51-81f3-3c8263f507d5">

Reply:
We still use it for not supported pairs
<img width="280" alt="image" src="https://github.com/novasamatech/nova-wallet-android/assets/107959809/69440a43-2a0b-4932-92fa-c7cdbc484db5">